### ISSUE TITLE
`--gene` option in identify want's to create strange directory. 

### ISSUE THREAD

**Issue by SilasK** [Ecogenomics/GTDBTk#399]
I want to try to run gtdb-tk on genes predicted with prodigal but I encounter an error. Probalby the intermediary directory is wrong and/or is not created. 

I also tried to input a batchfile, in which case the error is that the file doesn't exist, even if it does. 

Command:

```
gtdbtk identify --genes --genome_dir genomes/annotations/genes  --out_dir genomes/taxonomy/gtdb --extension FAA
```
stdout:
```

[2022-05-18 11:20:32] INFO: GTDB-Tk v2.1.0
[2022-05-18 11:20:32] INFO: gtdbtk identify --genes --genome_dir genomes/annotations/genes --out_dir genomes/taxonomy/gtdb --extension faa
[2022-05-18 11:20:32] INFO: Using GTDB-Tk reference data version r207: /srv/beegfs/scratch/shares/Trajkovski_Group/atlas_database/GTDB_V07
[2022-05-18 11:20:33] INFO: Identifying markers in 3 genomes with 1 threads.
[2022-05-18 11:20:33] INFO: Using supplied genomes as called genes, skipping Prodigal.
[2022-05-18 11:20:33] TASK: Identifying TIGRFAM protein families.
==> Processed 0/3 genomes (0%) |               | [?genome/s, ETA ?]
Error: Failed to open output file genomes/taxonomy/gtdb/identify/intermediate_results/marker_genes/MAG3.faa/MAG3.faa for writing

Process Process-1:
Traceback (most recent call last):
  File "/conda_envs/07efa96890606a0d27be0b3244a8bcbd/lib/python3.8/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/conda_envs/07efa96890606a0d27be0b3244a8bcbd/lib/python3.8/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/conda_envs/07efa96890606a0d27be0b3244a8bcbd/lib/python3.8/site-packages/gtdbtk/external/tigrfam_search.py", line 132, in _workerThread
    raise GTDBTkExit(f'Non-zero exit code returned when running hmsearch: {stdout}')
gtdbtk.exceptions.GTDBTkExit: Non-zero exit code returned when running hmsearch:
[2022-05-18 11:20:34] ERROR: An error was encountered while running hmmsearch.
[2022-05-18 11:20:34] ERROR: Controlled exit resulting from an unrecoverable error or warning.

```

## Environment
- [ ] Installed via pip (include the output of `pip list`)
- [x] Using a conda environment (include the output of `conda list && conda list --revisions`)
- [ ] Using a Docker container (include the `IMAGE ID` of the container)

## Server information
- CPU: `grep -m 1 "^model name" /proc/cpuinfo && grep -c "^processor" /proc/cpuinfo`
- RAM: `grep "^MemTotal" /proc/meminfo`
- OS: `cat /etc/os-release`

## Debugging information
- [x] `gtdbtk.log` has been included (drag and drop the file to upload).
- [x] Genomes have been included (if possible, and there are few).

## Additional comments

[Genome.zip](https://github.com/Ecogenomics/GTDBTk/files/8715005/Archive.zip)

**Comment by pchaumeil** [comment]
Hello,
Thanks for your feedback, 
we are working on a patch for 2.1 to fix this issue. it will be released in a few days.

Cheers,
Pierre

**Comment by alienzj** [comment]
I had the same problem today.

**Comment by pchaumeil** [comment]
Fixed in 2.1.1
